src/clients/mocks/MockEvents.ts

```diff
@@ -0,0 +1,76 @@
+import { utils as ethersUtils, Event, providers } from "ethers";
+import { random } from "lodash";
+
+const { id, keccak256, toUtf8Bytes } = ethersUtils;
+
+type Block = providers.Block;
+type TransactionResponse = providers.TransactionResponse;
+type TransactionReceipt = providers.TransactionReceipt;
```

Comment:
Additional change: imports were adjusted to be a little bit simpler (i.e. import parts of ethers, rather than the entire package).